app/talk/index.cjsx

```diff
@@ -8,21 +8,30 @@ TalkFootnote = require './footnote'
 module?.exports = React.createClass
   displayName: 'Talk'
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
+  logSearchBox: (searchItem) ->
```

Comment:
the method name and event type don't seem consistent? "search box" or "talk view" ?